src/main/java/seedu/address/logic/parser/ScheduleDateCommandParser.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import seedu.address.logic.commands.ScheduleDateCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ScheduleDateCommand object.
+ */
+public class ScheduleDateCommandParser implements Parser<ScheduleDateCommand> {
+
+    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("dd-MM-yyyy");
```

Comment:
I think in Daron's PR, he already added this DATE_FORMATTER into dateUtil. Can change to use his one after we merge his PR